Docs: Serverless function security considerations & Webhooks (#659)
redwoodjs/redwoodjs.com: docs/graphql.md

- dt has some thoughts on this
- insomnia -->

## Security

RedwoodJS already provides some best practices when deploying GraphQL to production, such as disabling the GraphQL Playground when you deploy and run in your production production environment.

> > Please come back soon where we will document more best practices when securing GraphQL as RedwoodJS reaches a v1.0 release candidate.

Author: I agree. I was struggling to say "coming soon". I'll reword and seen if maybe @thedavidprice can improve.

> RedwoodJS already provides some best practices when deploying GraphQL to production, such as disabling the GraphQL Playground when you deploy and run in your production production environment.
> 
> > Please come back soon where we will document more best practices when securing GraphQL as RedwoodJS reaches a v1.0 release candidate.

Reviewer: - there doesn't seem like there's enough here for two paragraphs; we could put it all in aside if you want
```suggestion
We'll document more GraphQL security best practices as Redwood reaches a v1.0 release candidate, but note that Redwood already bakes in some when deploying GraphQL to production, such as disabling the GraphQL Playground.
```

- dt has some thoughts on this
- insomnia -->

## Security

> We'll document more GraphQL security best practices as Redwood reaches a `v1.0` release candidate, but note that Redwood already has some baked-in when deploying GraphQL to production, such as disabling the GraphQL Playground. 

Reviewer 2: ```suggestion
We'll document more GraphQL security best practices as Redwood reaches a `v1.0` release candidate. For now, know that Redwood already has some baked-in best practices; for example, when deploying GraphQL to production, GraphQL Playground is automatically disabled. 
```

> Some of the biggest security improvements we'll be making revolve around Services (which are intimately linked to GraphQL since they're wrapped into your resolvers), and  for `v1.0` we plan on making all of your resolvers secure by default. You can even opt into this behavior now—see the [Secure Services](/docs/services#secure-services) section.

Reviewer 2: ```suggestion
Some of the biggest security improvements we'll be making revolve around Services (which are intimately linked to GraphQL since they're wrapped into your resolvers). For `v1.0` we plan to make all of your resolvers secure by default. You can even opt into this behavior now—see the [Secure Services](/docs/services) section.
```
